src/ring_theory/ideal/operations.lean

```diff
@@ -1604,3 +1611,91 @@ begin
 end
 
 end ring_hom
+
+namespace double_quot
+open ideal
+variables {R : Type u} [comm_ring R] (I J : ideal R)
+
+-- a few lemmas to help shorten the proofs later
+lemma left_proj_quot_add_mk (x :R) (hx : x ∈ I) : ideal.quotient.mk (I ⊔ J) x = 0 :=
+ideal.quotient.eq_zero_iff_mem.2 (ideal.mem_sup_left hx)
+
+/-- define `quot_left_to_quot_sum` to be the obvious ring hom `R/I → R/(I ⊔ J)` -/
+def quot_left_to_quot_sum : I.quotient →+* (I ⊔ J).quotient :=
+ideal.quotient.lift I (ideal.quotient.mk (I ⊔ J)) (left_proj_quot_add_mk I J)
```

Comment:
Is there an `ideal.quotient.map` that gives the ring morphism `R/I -> R/J` for `I < J`?

Reply:
I didn't find this lemma, so I added it to `ring_theory/ideals/basic` as `lift_mk_of_le`